### ISSUE TITLE
Feature Request: Flash button background on update

### ISSUE THREAD

**Issue by mintheredonethat** [krunkosaurus/bettertouchtool-crypto#6]
Not quite sure how apple scripting & BTT work, but based on the update interval, BTTTouchBarButtonColor should flash green/red depending on the price change (up/down) before returning back to the set BTTTouchBarButtonColor.

**Comment by krunkosaurus** [comment]
Yeah this could be cool, and better yet, perhaps a red or green outline based on its last 24 hour value — but there is no BTT options to do that, and that is what we build upon. If someone else can build it in, I can include it in the library as an option. :)
